maestro-client/src/main/java/maestro/Driver.kt

```diff
@@ -71,4 +71,10 @@ interface Driver {
     fun takeScreenshot(out: Sink)
 
     fun setLocation(latitude: Double, longitude: Double)
+
+    fun copyText(text: String)
```

Comment:
This shouldn't live in `Driver` as it is not device-specific. In fact, this should probably be isolated to `Orchestra`

Reply:
This applies to all 3 new methods

Reply:
There are 2 reasons i added it to the driver:
- Made testing easier
- Potential to expand functionality per device

With that said, i have no strong opinion on it.

Reply:
- I believe your current tests will cover this logic even if it would leave in `Orchestra`
- Let's add something to Driver only once we have a concrete use case where functionality could indeed be expanded